src/jenkins_job_insight/main.py

```
@@ -3490,10 +3549,58 @@ async def get_ai_configs_endpoint() -> list[dict]:
+
+
+@app.get("/metrics")
+async def prometheus_metrics() -> Response:
+    """Prometheus metrics endpoint."""
+    # Compute health_up from a lightweight health check
+    settings = get_settings()
+    db_path = str(storage.DB_PATH)
+    try:
+        health = await build_health_response(settings, db_path)
+        health_up = 0 if health["status"] == "unhealthy" else 1
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)